openapi3/schema.go

```diff
@@ -953,6 +964,13 @@ func (schema *Schema) visitJSONString(value string, fast bool) (err error) {
 				Reason:      cp.ErrReason,
 			}
 		}
+	} else if len(schema.Format) > 0 && cp == nil {
```

Comment:
Please open a separate PR for this as
> Tools that do not recognize a specific format MAY default back to the type alone, as if the format is not specified.

I'd prefer this functionality to be disable-able via a visit setting and I'd lean on unknown formats being allowed by default (thoughts about that default?)

Reply:
Again, this is not per-OpenAPIv3 IMO. Please move this to another PR (and add a new `SchemaValidationOption` "DisallowUnsupportedFormat()" maybe?)